src/index.html

```diff
@@ -1,12 +1,359 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="logo Moyo"
+          class="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__links is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__links" href="#">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__links" href="#">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__links"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__links" href="#">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__links" href="#">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__links" href="#">Photo</a>
+          </li>
+          
+          <li class="nav__item" >
+            <a class="nav__links last" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div class="container">
+        <div 
+          class="card card--1"
```

Comment:
not best idea to set number for card in classes

Reply:
https://github.com/mate-academy/layout_catalog/pull/3124/files#r1191157956
fix this comment

```suggestion
          class="card container__card"
```
looks like you don't use in styles this modifier. instead create element for container and move margins from .card block to .carc__container element (BEM)